spec/tapioca/gem/pipeline_spec.rb

```diff
@@ -1953,7 +1953,7 @@ module Toto; end
       RUBY
 
       output = template(<<~RBI)
-        Foo = T.let(T.unsafe(nil), ObjectSpace::WeakMap[T.untyped])
+        Foo = T.let(T.unsafe(nil), ObjectSpace::WeakMap)
```

Comment:
Can we add a test showing the behavior with the Sorbet version before and after the type of WeakMap was changed pease?

Reply:
Let's go with @Morriar's suggestion since we don't test with different sorbet versions. This  conditional will only enter sorbet version in CI. https://github.com/Shopify/tapioca/pull/1303#pullrequestreview-1216282219